scipy/optimize/tests/test_least_squares.py

```diff
@@ -369,6 +369,8 @@ def test_error_raised_when_all_tolerances_below_eps(self):
                       method=self.method, ftol=None, xtol=None, gtol=None)
 
     def test_convergence_with_only_one_tolerance_enabled(self):
+        if self.method == 'lm':
+            return # should not do test
```

Comment:
Add another space before the comment - style checker requirement.

Reply:
Oh, sorry. I fixed it.